# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/nsLayoutUtils.cpp
# Commit: 0e259884f052
# Full Hash: 0e259884f052230b24601efc3cde3a5ef5d4e6ad
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2019-05-01 15:54:31
# Regressor Bug: 1535945
# File Overlap Count: 1
# Description:
#   Bug 1535945 - Don't skip invalidating frames when creating displayports for async scrollable ancestors. r=tnikkel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D23816
# ==============================================================================

diff -r 732976d3f555 -r 0e259884f052 layout/base/nsLayoutUtils.cpp
--- a/layout/base/nsLayoutUtils.cpp	Wed May 01 08:16:44 2019 +0000
+++ b/layout/base/nsLayoutUtils.cpp	Wed May 01 04:23:07 2019 +0000
@@ -3346,7 +3346,7 @@
 }
 
 void nsLayoutUtils::SetZeroMarginDisplayPortOnAsyncScrollableAncestors(
-    nsIFrame* aFrame, RepaintMode aRepaintMode) {
+    nsIFrame* aFrame) {
   nsIFrame* frame = aFrame;
   while (frame) {
     frame = nsLayoutUtils::GetCrossDocParentFrame(frame);
@@ -3365,7 +3365,7 @@
         !nsLayoutUtils::HasDisplayPort(frame->GetContent())) {
       nsLayoutUtils::SetDisplayPortMargins(frame->GetContent(),
                                            frame->PresShell(), ScreenMargin(),
-                                           0, aRepaintMode);
+                                           0, RepaintMode::Repaint);
     }
   }
 }